--- origsrc/htmldoc-1.9.x-r1703/configure.ac	2011-12-23 00:54:00.000000000 -0600
+++ src/htmldoc-1.9.x-r1703/configure.ac	2012-03-15 23:48:00.237113100 -0500
@@ -170,7 +170,7 @@ AC_CHECK_HEADER(openssl/ssl.h,
 	AC_CHECK_LIB(ssl,SSL_new,
 	    [SSLFLAGS="-DOPENSSL_DISABLE_OLD_DES_SUPPORT -DOPENSSL_NO_KRB5"
 	     SSLLIBS="-lssl $libcrypto"
-	     AC_DEFINE(HAVE_SSL)],,
+	     AC_DEFINE(HAVE_LIBSSL)],,
 	    $libcrypto)
 
 	if test "x${SSLLIBS}" != "x"; then
